src/videosubscribecontext/VideoSubscribeContext.ts

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import VideoStreamIdSet from '../videostreamidset/VideoStreamIdSet';
+import VideoStreamIndex from '../videostreamindex/VideoStreamIndex';
+
+export default interface VideoSubscribeContext {
```

Comment:
- nit: Recommend matching the method order bewteen interface and implementation
  e.g.
  1. clone get functions
  2. ref get functions
  3. set functions
  4. wantsReceiveVideo()
  5. clone()

- If I were to rewrite this class in V2, I would state the "cloned" version more explicitly. Don't worry about this yet.
```
getClonedVideoStreamIndex();

// Typical get and set reference
getVideoStreamIndex();
setVideoStreamIndex(videoStreamIndex: VideoStreamIndex)
```